src/StyledComponents.js

```diff
@@ -0,0 +1,75 @@
+import { styled } from "@mui/material/styles";
+import Paper from "@mui/material/Paper";
+import customTheme from "./theme";
+import { Box, Button, Divider, Link } from "@mui/material";
+
+const ContactItem = styled(Paper)(({ theme }) => ({
+  padding: theme.spacing(1),
+  textAlign: "center",
+}));
+export default ContactItem;
```

Comment:
Let's keep export in the same line, will be much easier to read and more consistent 
`export const CustomButton = styled(Button)(({ theme }) => ({
  backgroundColor: theme.palette.secondary.main,
  color: theme.palette.common.black,
  '&:hover': {
    backgroundColor: theme.palette.primary.main,
  },
}));`

Reply:
done